samples/browser/src/main/res/xml/data_extraction_rules.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<data-extraction-rules>
+    <cloud-backup>
+        <include domain="sharedpref" path="."/>
```

Comment:
```suggestion
        <exclude domain="sharedpref" />
        <exclude domain="database" />
```

For the sake of the argument, our real apps wouldn't want to include shared prefs in a sync. This might cause conflicts when a new device is added to Sync or preferences that hold no-op values (onboarding, etc).

Should we make the sample do the same as a reference for them?

Reply:
I only added        ` <include domain="sharedpref" path="."/>` because there already was a `backup_rules` file that contains   `  <include domain="sharedpref" path="."/>` for sample browser. And I kept the same settings as before for all samples. We can set them either way, I do not think anyone uses them as a guide anymore.

Reply:
Oh I see, I must have missed that. Yeah then there is no need to change them for now.